### DISCUSSION TITLE
Difficulty installing GS_M-2.2.0 on intel-based MacBook Air

### DISCUSSION THREAD

**Discussion by mjpeeters** [G-String-Legacy/G_String#23] [General]
Greetings,
I am trying (if possible) to install "GS_M-2.2.0" on my intel-chip MacBook Air. I followed the directions for downloading to bypass Apple Gateway. Do I still need to have download it on a non-Apple device? I have attached a screen-shot to show where my issue is.
Thank you
Michael

![Screenshot 2024-03-04 at 14 31 00](https://github.com/G-String-Legacy/G_String/assets/162215436/fb35f790-6175-4318-89bc-00bcc554db52)

**Comment by Papa-26** [comment]
Hello Michael.

Version 2.2.0 is built in Java 21, which is not supported for mac Intel, only Silicon. I recommend GS_M-2.0.12 built with java 17. You can use the terminal method from https://www.youtube.com/watch?v=lxfD9mm2D2I. Then you can download it directly to your MacBook Air.

Ralph

p.s. I am past my own LTS. So I don't know, how long I can continue supporting G-String. Please start using the discussion forum. Steven Anderson([stevenarild@gmail.com](https://github.com/orgs/G-String-Legacy/discussions/stevenarild@gmail.com)) Copenhagen Hearing and Balance Center, Dept. of Otorhinolaryngology—Head & Neck Surgery, Rigshospitalet, Copenhagen, Denmark is now the 'owner' of the G-String repository.

**Comment by mjpeeters** [comment]
Thank you for your confirmation, Ralph. I will get a new Apple-silicone MBA later this year

I had a copy of GS_V and was able to use that successfully for my analysis.

Michael

(BTW, this is a posting in 'Discussion' ...so I am confused by your PS comment)
